docs/rancher/index.md

```diff
@@ -0,0 +1,191 @@
+# Rancher Cookbook
+
+Recipe for installing
+Sextant | Community edition on a Rancher managed kubernetes cluster to deploy
+and manage blockchain networks.
+
+## Install Prerequisite Tools
```

Comment:
Make this section "Prerequisites" please

Reply:
In addition to the note about kubectl please add words to the effect that a prerequisite is SUSE Rancher v2.6 or above

Reply:
Please include link to SUSE Rancher https://www.suse.com/products/suse-rancher/ for newbies

Reply:
Done, I've specified versions for both Kubernetes and Rancher